"panic: inappropriate ioctl for device" when output is sent to a shell pipeline

```console
$ gh --version
gh version 2.20.2 (2022-11-15)
https://github.com/cli/cli/releases/tag/v2.20.2
$ gh actions-status trussworks | tee output
panic: inappropriate ioctl for device

goroutine 1 [running]:
main.getTerminalWidth(0xc00007a7e0)
        /home/runner/work/gh-actions-status/gh-actions-status/main.go:103 +0xc5
main._main(0xc00007a7e0, 0x0, 0x0)
        /home/runner/work/gh-actions-status/gh-actions-status/main.go:172 +0x9b
main.main()
        /home/runner/work/gh-actions-status/gh-actions-status/main.go:449 +0xc5
$
```

--- a/main.go
+++ b/main.go
@@ -97,6 +97,10 @@ func truncateWorkflowName(name string, length int) string {
 }
 
 func getTerminalWidth() int {
+	if !term.IsTerminal(int(os.Stdout.Fd())) {
+		return 80
+	}
+
 	width, _, err := term.GetSize(int(os.Stdout.Fd()))
 
 	if err != nil {

